src/AaveV3AvaxretrofundingPayload_20230626/AaveV3AvaxretrofundingPayload_20230626_Test.t.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import 'forge-std/Test.sol';
+
+import {AaveV3AvaxretrofundingPayload} from './AaveV3AvaxretrofundingPayload_20230626.sol';
+import {AaveV3Avalanche} from 'aave-address-book/AaveV3Avalanche.sol';
+import {GovHelpers, TestWithExecutor} from 'aave-helpers/GovHelpers.sol';
```

Comment:
TestWithExecutor is not part of GovHelpers are you trying to use `ProtocolV3TestBase` instead?